### PR TITLE
Make minor updates, adapt to some new changes

### PR DESCRIPTION
**Pull request by harendra-kumar**

### CODE REVIEW

File: docs/User/ProjectRelated/Upgrading.md [hunk 1]
```diff
@@ -88,7 +91,6 @@ Explicit stream fold functions have been omitted from the new stream
 module. You can use the following equivalent definitions:
 
 ```haskell docspec
->>> toList = Stream.fold Fold.toList
```
Comment (adithyaov) [inline docs/User/ProjectRelated/Upgrading.md:91]: Did you add back `toList`?

File: docs/User/ProjectRelated/Upgrading.md [hunk 2]
```diff
@@ -106,23 +108,21 @@ module. You can use the following equivalent definitions:
 >>> length = Stream.fold Fold.length
 >>> last = Stream.fold Fold.latest
 >>> head = Stream.fold Fold.one
->>> foldr f a = Stream.fold (Fold.foldr' f a)
```
Comment (adithyaov) [inline docs/User/ProjectRelated/Upgrading.md:109]: Did you add this back?

**Review by adithyaov — APPROVED** [review]

File: docs/User/ProjectRelated/Upgrading.md (hunk 1 above)
Comment (harendra-kumar) [reply docs/User/ProjectRelated/Upgrading.md:91]: Yes.

File: docs/User/ProjectRelated/Upgrading.md (hunk 2 above)
Comment (harendra-kumar) [reply docs/User/ProjectRelated/Upgrading.md:109]: Yes, foldr and foldrM.

**Pull request merged**